stripe-ui-core/src/main/java/com/stripe/android/uicore/format/CurrencyFormatter.kt

```diff
@@ -13,7 +13,7 @@ object CurrencyFormatter {
     private const val MAJOR_UNIT_BASE = 10.0
     private val SERVER_DECIMAL_DIGITS =
         mapOf(
-            setOf("UGX", "AFN", "ALL", "AMD", "COP", "IDR", "ISK", "PKR", "LBP") to 2
+            setOf("UGX", "AFN", "ALL", "AMD", "COP", "IDR", "ISK", "PKR", "LBP", "MMK") to 2
```

Comment:
Before this change, we used `currency.defaultFractionDigits`, which returned 0 for MMK. As a result, we divided the amount by `10 ^ 0 = 1`.

Now, we override `defaultFractionDigits` and divide the amount by `10 ^ 2 = 100`, resulting in the correct two-decimal formatting.